assets/sass/components/_links.scss

```diff
@@ -211,7 +211,8 @@ Style guide: Link styles.1 Hover links
   }
 }
 
-main > article {
+header[role='banner'],
+article {
```

Comment:
(Not sure where my comment went)…

@petronbot I think we should change `article` to `main` here?